### DISCUSSION TITLE
Please share config about cyberpunk background

### DISCUSSION THREAD

**Discussion by IvayloHrTonev** [wezterm/wezterm#4807] [General]
Hi everyone,

I would like to ask @wez to share (if possible) the config that generates this image:

https://wezfurlong.org/wezterm/screenshots/wezterm-vday-screenshot.png

That would be great and I would be sincerely grateful :)

**Comment by wez** [comment]
See https://www.reddit.com/r/neovim/comments/16lwnga/comment/k2o531h/
The image can be found with a search engine: https://duckduckgo.com/?t=ffab&q=vday+cyberpunk&iax=images&ia=images

**Reply by wez** [reply]
The font config for wezterm can be found in https://wezfurlong.org/wezterm/config/lua/config/font_rules.html

**Reply by IvayloHrTonev** [reply]
Thank you, @wez for your detailed reply
